internal/proto/controller/auth/v1/auth.proto

```diff
@@ -51,4 +51,7 @@ message RequestInfo {
 
   // the client ip for the request
   string client_ip = 140;
+
+  // url actions from the request path
+  repeated string actions = 141;
```

Comment:
Ditto is this required?

Reply:
This is required to set the actions before https://github.com/hashicorp/boundary/blob/96688b36d577a8769f823e346f52d7e63abab2ac/internal/daemon/controller/handler.go#L652 interceptor trims the `:callback` action.
For example, if we get a request like `http://127.0.0.1:9200/v1/auth-methods/ampw_1234567890:authenticate:callback`, when it gets to the service the path will be `http://127.0.0.1:9200/v1/auth-methods/ampw_1234567890:authenticate` because `:callback` is trimmed. This field basically captures all actions before changes are made to the url path